nbodykit/plugins/datasource/ZeldovichSim.py

```diff
@@ -0,0 +1,96 @@
+from nbodykit.extensionpoints import DataSource
+from nbodykit import utils
+
+import numpy
+import logging
+
+logger = logging.getLogger('ZeldovichSim')
+         
+class ZeldovichSimDataSource(DataSource):
```

Comment:
I crafted a 'proper' C++ binding in around 16 hours. It wouldn't be too much more difficult to do something like this with Cython.